tests/TestCodeGen.hs

```diff
@@ -41,18 +41,20 @@ codeGenTests = testGroup "Code generator unit tests"
   , camelCaseMessageFieldNames
   , don'tAlterEnumFieldNames
   , knownTypeMessages
+  , pythonInteroperation
+  ]
+
+pythonInteroperation :: TestTree
+pythonInteroperation = testGroup "Python interoperation" $ do
 #ifdef LARGE_RECORDS
-  , simpleEncodeDotProto LargeRecords "Binary"
-  , simpleDecodeDotProto LargeRecords "Binary"
-  , simpleEncodeDotProto LargeRecords "Jsonpb"
-  , simpleDecodeDotProto LargeRecords "Jsonpb"
+  recStyle <- [RegularRecords, LargeRecords]
 #else
-  , simpleEncodeDotProto RegularRecords "Binary"
-  , simpleDecodeDotProto RegularRecords "Binary"
-  , simpleEncodeDotProto RegularRecords "Jsonpb"
-  , simpleDecodeDotProto RegularRecords "Jsonpb"
+  recStyle <- [RegularRecords]
 #endif
-  ]
+  tt <- ["Data.Text.Lazy.Text", "Data.Text.Text", "Data.Text.Short.ShortText"]
+  format <- ["Binary", "Jsonpb"]
+  direction <- [simpleEncodeDotProto, simpleDecodeDotProto]
+  [direction recStyle tt format]
```

Comment:
I would prefer using pure vs. list syntax. It's a minor difference but it fits into the "schema" of do blocks more than the list syntax does if that makes sense.

Reply:
Here I was thinking it would make it clear which `Monad` is in play here.  But OK.

Reply:
Changed to `pure @[]` in babd2a8195abb03f2b9180da83d15410eea7fcc3.

Reply:
That's a good choice